net/bind/Makefile

```diff
@@ -122,41 +127,30 @@ export BUILD_CC="$(TARGET_CC)"
 TARGET_LDFLAGS += -Wl,--gc-sections,--as-needed
 
 CONFIGURE_ARGS += \
-	--enable-ipv6=$(if $(CONFIG_IPV6),yes,no) \
-	--with-randomdev="/dev/urandom" \
-	--disable-threads \
 	--disable-linux-caps \
 	--with-openssl="$(STAGING_DIR)/usr" \
 	--with-libtool \
 	--without-lmdb \
-	--enable-epoll=yes \
-	--with-gost=no \
-	--with-gssapi=no \
-	--with-ecdsa=$(if $(CONFIG_OPENSSL_WITH_EC),yes,no) \
-	--with-eddsa=$(if $(CONFIG_BIND_ENABLE_EDDSA),yes,no) \
```

Comment:
Where did these options go? Did bind drop support for these features, or are they enabled unconditionally now, or what? I haven't been able to find anything in the release notes to clarify this, but I'd like to understand the change before we merge this.

Reply:
ECDSA support in OpenSSL is mandatory. Other options are nowhere to be found (Do not get recognized by `./configure`)

Reply:
Looks like the check for EDDSA and others are implicit, and can't be disabled. So this is fine.